submissions/chizaramOpara/HTML/Shopnownow.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset= "UTF8">
+	<title>Shopnownow</title>
+	<head>
+	<body>  
+	<header><b> SHOPNOWNOW NG <b><header>
+	<nav>
+	<em>Shopping Made Easier<em>
+	<p> <b> Hurray! Our Christmas sales is ongoing </b> </p>
+	<nav>
+	<img src="C:/Users/user/Pictures/shopnownow.jpg" width="150"/>
+	</nav>
+	<a href= "#">
+	<h2> Bone Straight </h2>
+	</a>
+	<p><em> Luxury hair that will have you looking all glammed for the gram. You definitely need a bonestraight to slay this Christmas </em></p>
+	<nav>
+	<img src="C:/Users/user/Pictures/bone straight.jpg" width="150"/>
+	</nav>
+	<p> $2,000 </p>
+	<a href= "#">
+	<h2> Birkin </h2>
+	</a>
+	<p><em>Looking for a bag that says bougie, classy and chick? Shop Birkin </em></p>
+	<nav>
+	<img src="C:/Users/user/Pictures/birkin.jpg" width="150"/>
+	</nav>
+	</a>
+	<nav>
+	<p> $3,000 </p>
+	</nav>
+	<section><h1> Similar products <h1></section>
+	<ol> 
+	<li> 
+	<tr> Hermes $5000</tr> 
+	</li>
+	<li>
+	<tr> Scandinavian Virgin Hair $10000</tr> 
+	</li>
+	<li>
+	<tr> Luis Vuitton $3500</tr> 
+	</li>
+	</ol>
+	<page-break-before/>
+		<footer>
+		Stay Trendy with Us.
+		</footer>
+	<body>
+	
+	
```

Comment:
This file is not properly formatted. Also, the sections ought to be within the body. Ensure you close and open the tags appropriately as well.